scipy/stats/_continuous_distns.py

```diff
@@ -10481,12 +10481,20 @@ class tukeylambda_gen(rv_continuous):
     %(example)s
 
     """
+    _support_mask = rv_continuous._open_support_mask
```

Comment:
~~I looked up what this does. If I'm thinking straight, this should only have an effect on the output of the `pdf` method, right? And if I can extrapolate from the picture on Wikipedia, it looks like the PDF at `x=a` and `x=b` will always be zero, so unless the PDF was not returning the right answer at the endpoints, this would not have an effect. Did you change this just to reduce the calculation for these special cases, or does it fix something?~~ I see you did this to avoid a warning.